builder/softlayer/step_capture_image.go

```diff
@@ -40,6 +86,33 @@ func (self *stepCaptureImage) Run(state multistep.StateBag) multistep.StepAction
 		return multistep.ActionHalt
 	}
 
+	if config.ImageType == "standard" {
```

Comment:
Interesting idea! I didn't think that the image shows up in the list of images until image creation has finished. I'll test it out and if it works I'll move it before waiting for image creation to finish.

Ok, I'll move it to a separate function, but I won't be able to add tests for that function as it makes API calls. Later on we can add some stubs to make testing possible, and go back and write the tests for this function.